src/Pages/Main/ArticleMd/ArticleMd.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import Card from '../Component/Card';
+import './ArticleMd.scss';
+class ArticleMd extends Component {
+  constructor() {
+    super();
+    this.state = { cardData: [] };
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/Card.json', {
+      method: 'GET',
+    })
```

Comment:
```suggestion
    fetch('/data/Card.json')
```
- get은 디폴트 값입니다. 생략 가능합니다!
- public 경로에서 가져올 경우 주소와 포트를 생략할 수 있습니다.

Reply:
오..감사합니다 도현님!! 수정했습니다.

Reply:
@2wo0 저 종택님입니다....